vllm/model_executor/model_loader/loader.py

```diff
@@ -1005,16 +1007,21 @@ def _unquantized_generator(self, hf_weights_files, use_safetensors,
             if any(target_module in weight_name for target_module in
                    self.target_modules) and weight_name.endswith(".weight"):
                 weight_name = weight_name.replace(".weight", ".qweight")
```

Comment:
A general questions I have about BNB is why do we use `.qweight` in the [BNBLinearMethod](https://github.com/vllm-project/vllm/blob/74b529ceeead8d4b44ded858f7c28bca9c1629ba/vllm/model_executor/layers/quantization/bitsandbytes.py#L207) but the model checkpoints actually use `.weight`? It seems we could avoid some logic by having the quant method directly use `.weight`
<img width="879" alt="Screenshot 2024-11-02 at 2 14 23 PM" src="https://github.com/user-attachments/assets/9dfc6635-4a10-45d3-a8f7-198b5e1dc548">

Reply:
Perhaps it's to maintain consistency with other quantization algorithms like GPTQ implementations see: https://github.com/vllm-project/vllm/blob/74b529ceeead8d4b44ded858f7c28bca9c1629ba/vllm/model_executor/layers/quantization/gptq.py#L207

Reply:
Okay, let's try to remove this in the future if possible! The "consistency" is just coincidental - we usually aim for parameters to have the same name as in the checkpoint format to make weight loading simple